src/applications/claims-status/components/StemClaimListItemV3.jsx

```diff
@@ -8,10 +8,9 @@ export default function StemClaimListItemV3({ claim }) {
   if (!claim.attributes.automatedDenial) {
     return null;
   }
-  const formattedDeniedAtDate = () =>
```

Comment:
This component was throwing an error because both `formattedDeniedAtDate` and `formattedReceiptDate` are functions, not normal values and React was unhappy about that